tests/phpunit/integration/tests/REST_API/Stories_Users_Controller.php

```diff
@@ -123,7 +123,7 @@ public function test_filter_user_query_capabilities_query_supported(): void {
 		);
 	}
 
-	public function filter_rest_user_collection_params( $query_params ) {
+	public function filter_rest_user_collection_params( array $query_params ): array {
```

Comment:
This runs throught a filter right? Thought we were not adding type hints to filter callbacks?